src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -866,6 +866,18 @@ InferenceProfiler::ValidLatencyMeasurement(
     }
   }
 
+  const double minimum_valid_requests_percent = 0.5;
+  if (verbose_ &&
+      valid_latencies->size() / static_cast<double>(timestamps.size()) <
```

Comment:
I understand the operator precedence is correct, but having the second condition in paranthesis will be better for readability.

Reply:
Added parentheses 👍